cni/network/invoker.go

```diff
@@ -27,8 +27,18 @@ type IPAMAddConfig struct {
 }
 
 type IPAMAddResult struct {
-	ipv4Result       *cniTypesCurr.Result
-	ipv6Result       *cniTypesCurr.Result
+	// Splitting defaultInterfaceInfo from secondaryInterfaceInfo so we don't need to loop for default CNI result every time
```

Comment:
loop for defaultInterfaceInfo every time?

Reply:
yeah like whenever we refer to defaultInterfaceInfo, we'd have to loop through interfacesInfo (if we don't split) and determine based on NICType or something

Reply:
i asked you to update comment from default CNI result to defaultInterfaceInfo